src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/DJApplicantReceivedNotificationHandler.java

```diff
@@ -106,14 +115,27 @@ private CallbackResponse notifyApplicantSolicitorDefaultJudgmentReceived(Callbac
                 String.format(templateReference, caseData.getLegacyCaseReference())
             );
         }
-        if (ofNullable(caseData.getRespondent2()).isEmpty()) {
+        if (ofNullable(caseData.getRespondent2()).isEmpty() && !caseData.isLipvLipOneVOne()) {
             notificationService.sendMail(
                 caseData.getApplicantSolicitor1UserDetails().getEmail(),
                 identifyTemplate(caseData),
                 addProperties(caseData),
                 String.format(templateReference, caseData.getLegacyCaseReference())
             );
         }
+
+        if (caseData.isLipvLipOneVOne()
+            && toggleService.isLipVLipEnabled()
+            && ofNullable(caseData.getRespondent2()).isEmpty()) {
```

Comment:
no need of 3rd validation as that will be always false if its LiPvsLip claim

Reply:
Done